schema.sql

```diff
@@ -1,11 +1,29 @@
 /* Database schema to keep the structure of entire database. */
 CREATE TABLE animals (
-	user_id integer SERIAL PRIMARY KEY,
+	id integer SERIAL PRIMARY KEY,
 	name VARCHAR,
 	date_of_birth date,
 	escape_attempts integer,
 	neutered bool,
 	weight_kg DECIMAL
 );
 
-ALTER TABLE animals ADD species VARCHAR;
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR;
```

Comment:
- [ ] I think instead of adding this species column to the animals' table, you are expected to remove it because you are going to create a separate table.
Therefore, I would suggest you run the command 
```sql
ALTER TABLE animals REMOVE species;
```

and add that command here in the schema.sql 😉

Reply:
I just forget to add this query to the schema.sql. However, it was already implemented correctly, please.